Ui/Component/Listing/Column/SyncedCreditmemo.php

```diff
@@ -7,6 +7,7 @@
 use \Magento\Ui\Component\Listing\Columns\Column;
 use \Magento\Framework\Api\SearchCriteriaBuilder;
 use \Magento\Framework\Stdlib\DateTime\Timezone;
+use \Magento\Framework\Exception\NoSuchEntityException;
 
 class SyncedCreditmemo extends Column
```

Comment:
Do we need to set the logger filename in this constructor as well?

Reply:
Yes we need the same includes/constructor statements used in SyncedOrder.